core/src/main/java/com/sequenceiq/distrox/v1/distrox/StackOperations.java

```diff
@@ -346,4 +346,5 @@ private void validateAccessDto(StackAccessDto dto) {
             throw new BadRequestException("A stack name or crn must be provided. One and only one of them.");
         }
     }
+
```

Comment:
The only change in this file is this blank line, I would exclude it from the changes.

Reply:
It just hurts my eyes when the last method has no trailing enter :)